Dockerfile

```diff
@@ -0,0 +1,12 @@
+FROM ubuntu:18.04
```

Comment:
Any reason not to use 20.04 LTS?

Reply:
This is just to keep in sync with CI and make sure that we can install older versions of TensorFlow that might not support Python 3.8